runtime/doc/lua.txt

```diff
@@ -3034,6 +2984,473 @@ vim.glob.to_lpeg({pattern})                               *vim.glob.to_lpeg()*
         pattern
 
 
+==============================================================================
+VIM.LPEG                                                            *vim.lpeg*
+
+
+The Lpeg library for parsing expression grammars is being included as
+`vim.lpeg` (https://www.inf.puc-rio.br/~roberto/lpeg/).
+
+In addition, its regex-like interface is available as
+|vim.re| (https://www.inf.puc-rio.br/~roberto/lpeg/re.html).
+
+vim.lpeg.B({pattern})                                           *vim.lpeg.B()*
+    Returns a pattern that matches only if the input string at the current
+    position is preceded by `patt`. Pattern `patt` must match only strings
+    with some fixed length, and it cannot contain captures. Like the and
+    predicate, this pattern never consumes any input, independently of success
+    or failure.
+
+    Parameters: ~
+      • {pattern}  (`vim.lpeg.Pattern`)
+
+    Return: ~
+        (`vim.lpeg.Pattern`)
+
+vim.lpeg.C({patt})                                              *vim.lpeg.C()*
+    Creates a simple capture, which captures the substring of the subject that
+    matches `patt`. The captured value is a string. If `patt` has other
+    captures, their values are returned after this one.
+
+    Example:>lua
+        local function split (s, sep)
+          sep = lpeg.P(sep)
+          local elem = lpeg.C((1 - sep)^0)
+          local p = elemelem)^0                                       * (sep *
```

Comment:
luadoc parser should be fixed (there are many similar occurrences, e.g., `Cf()`, `Cp()`, `V`)

Reply:
Yes `* sep (*` tag... from `---   local p = elem * (sep * elem)^0
`; doxygen is not aware of the markdown block.

Wrapping these with `<pre></pre>` also doesn't work. Not sure yet how to fix this.

Reply:
Fixed the bug in `gen_vimdoc.py`